docker-compose.yml

```diff
@@ -76,6 +76,7 @@ services:
     environment:
       NB_API_QUERY_URL: ${NB_API_QUERY_URL}
```

Comment:
A thought: I'm realizing that the name `NB_API_QUERY_URL` is pretty confusing and ambiguous in terms of whether it points to the query tool or API, now that we also have `NB_QUERY_URL_PATH`.

We should probably consider renaming `NB_API_QUERY_URL` to sth clearer, either as part of https://github.com/neurobagel/recipes/issues/30 or the issue concerning query tool -> f-API or query tool -> n-API. Maybe you can link this PR there, or vice versa, once the issue is open? :)

Reply:
I think it's better to be addressed in #30 but we can also discuss it in https://github.com/neurobagel/query-tool/issues/225